src/components/Questionnaire/QuestionTypes/EncounterQuestion.tsx

```
@@ -239,33 +239,51 @@ export function EncounterQuestion({
   }
-
+  const isDischarged = encounter.status === EncounterStatus.DISCHARGED;
   return (
     <div className="space-y-6">
       <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
         {/* Basic Details */}
         <div className="space-y-2">
           <Label>{t("encounter_status")}</Label>
+          <div
+            data-testid="encounter-status"
+            role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)